src/librustc/ty/print/pretty.rs

```diff
@@ -460,6 +460,19 @@ pub trait PrettyPrinter<'tcx>:
         })
     }
 
+    fn print_type_ascribed(
```

Comment:
So there's two issues here:
1. I should've said `type_ascribed_value` instead of `type_ascribed`, maybe?
2. I really did mean without the `print_` prefix, and another closure instead of `Ty` for the second part, and placing this wherever `generic_delimiters` is (i.e. this is a formatting helper, not printing anything specific)

Reply:
Done. It was already in the same impl block as `generic_delimiters`, but I moved it closer.